functions/api/category.js

```diff
@@ -284,10 +297,13 @@ module.exports.addOrUpdateCategory = functions.firestore
     .document(`/${constants.CATEGORIES}/{categoryName}`)
     .onWrite(async (change, context) => {
         const categoryName = context.params.categoryName
-        if (!change.after._fieldsProto) {
-            logger.debug(`Category ${change.before.id} has been deleted`)
+        if (!change.before._fieldsProto) {
```

Comment:
do you need this if condition

Reply:
I don't. It's just for the log. And yes, I do need to update these triggers. I will work on it in another branch.